test/e2e/tests/api-usage/account-tracker-api-usage.spec.ts

```diff
@@ -0,0 +1,202 @@
+      request.url.match('infura'),
+    );
+
+    for (const r of seenProviderRequests) {
+      const json = await r.body.getJson();
+      if (json !== undefined) {
+        allInfuraJsonRpcRequests.push(json);
+      }
+    }
+  }
+
+  return allInfuraJsonRpcRequests;
+}
+
+function getSpecifiedJsonRpcRequests(
+  jsonRpcRequestArray: JsonRpcRequest[],
+  methodsToGet: string[],
+) {
+  return jsonRpcRequestArray.filter(({ method }) =>
+    methodsToGet.includes(method),
+  );
+}
+
+describe('Account Tracker API Usage', function () {
+  it('should not make eth_call or eth_getBalance requests before the UI is opened and should make those requests after the UI is opened', async function () {
+    const RPC_METHODS_TO_TEST = ['eth_call', 'eth_getBalance'];
+
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().withNetworkControllerOnMainnet().build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test?.fullTitle(),
+        testSpecificMock: mockInfura,
+      },
+      async ({
+        driver,
+        mockedEndpoint,
+      }: {
+        driver: Driver;
+        mockedEndpoint: MockedEndpoint[];
+      }) => {
+        await driver.delay(3000);
+        let allInfuraJsonRpcRequests = await getAllInfuraJsonRpcRequests(
+          mockedEndpoint,
+        );
+        let ethCallAndGetBalanceRequests = getSpecifiedJsonRpcRequests(
+          allInfuraJsonRpcRequests,
+          RPC_METHODS_TO_TEST,
+        );
+
+        assert.ok(
+          ethCallAndGetBalanceRequests.length === 0,
+          `An ${RPC_METHODS_TO_TEST.join(
+            ' or ',
+          )} request has been made to infura before opening the UI`,
+        );
+
+        await unlockWallet(driver);
+        await driver.delay(3000);
+
+        allInfuraJsonRpcRequests = await getAllInfuraJsonRpcRequests(
+          mockedEndpoint,
+        );
+        ethCallAndGetBalanceRequests = getSpecifiedJsonRpcRequests(
+          allInfuraJsonRpcRequests,
+          RPC_METHODS_TO_TEST,
+        );
+
+        assert.ok(
+          ethCallAndGetBalanceRequests.length > 0,
+          `No ${RPC_METHODS_TO_TEST.join(
+            ' or ',
+          )} request has been made to infura since opening the UI`,
+        );
+      },
+    );
+  });
+
+  it('should not make eth_call or eth_getBalance requests after the UI is closed', async function () {
+    const RPC_METHODS_TO_TEST = [
+      'eth_getBlockByNumber',
+      'eth_call',
+      'eth_getBalance',
+    ];
+
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().withNetworkControllerOnMainnet().build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test?.fullTitle(),
+        testSpecificMock: mockInfura,
+      },
+      async ({
+        driver,
+        mockedEndpoint,
+      }: {
+        driver: Driver;
+        mockedEndpoint: MockedEndpoint[];
+      }) => {
+        await unlockWallet(driver);
+        await driver.delay(3000);
```

Comment:
Nit: can we use predefined delay here 
```suggestion
        await driver.delay(veryLargeDelayMs);
```